migrate-configuration-v2.rst

```diff
@@ -15,7 +15,7 @@ for all projects in order to build documentation successfully.
 We will stop creating a default configuration file *behind the scenes*, making assumptions for your projects,
 and automatically fixing problems with project configurations.
 
-We plan to deploy these changes and **start failing builds without a configuration file in September 2023**.
+We plan to deploy these changes and **start failing builds without a configuration file on September 20, 2023 at 12:01am UTC**.
```

Comment:
A quick note: We can use "brownout" internally, but it's not gonna make much sense in a public announcement :)

Reply:
Brownout is a term fairly commonly used here, but probably worth using some supporting language here as well. Here is a GitHub announcement for comparison:
https://github.blog/changelog/2021-08-10-brownout-notice-api-authentication-via-query-parameters-for-48-hours/

Also noting, this was the third brownout and it lasted 48h. I wonder if instead of 2h blocks we instead just do something like a 12h period (July 24) and a 24h period (Aug 7).

Reply:
I see! allow me to revise that: We cannot use brownout without accompanying it with a good description of what it means :)